src/components/Input/Input.scss

```diff
@@ -83,40 +87,36 @@
         background-color: inherit;
         color: inherit;
         box-sizing: border-box;
+        composes: OneUI-label-text from global;
 
         &--type_search::-webkit-search-cancel-button {
             display: none;
```

Comment:
doesn't `appearance: none` works better? I should check it in all browsers, did you try them?